tests/acceptance/test_emails.py

```diff
@@ -36,9 +37,17 @@
 
 
 def read_txt_email_fixture(name):
+    # XXX(python3): We have different fixtures for python2 vs python3 tests
```

Comment:
We can easily replace this `random` and `seed` logic with `uuid` and keep getting consistent "random" numbers across python versions (and probably even across languages):

```
byk@DESKTOP-QII4CIG:$ python
Python 2.7.17 (default, Nov  7 2019, 10:07:09)
[GCC 7.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc2', version=4)
UUID('61626364-6162-4364-a162-636461626332')
byk@DESKTOP-QII4CIG:$ logout
[byk@DESKTOP-QII4CIG ~/P/g/sentry]$ python                                                                                                        Python 3.8.6rc1 (tags/v3.8.6rc1:08bd63d, Sep  7 2020, 23:10:23) [MSC v.1927 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> uuid.UUID(bytes=b'abcdabcdabcdabc1', version=4)
UUID('61626364-6162-4364-a162-636461626331')
>>> uuid.UUID(bytes=b'abcdabcdabcdabc2', version=4)
UUID('61626364-6162-4364-a162-636461626332')
```

Reply:
Turns out we have an easier way out if this works. `random.seed()` has a new `version` parameter which should produce the same values with the PY2 implementation when `version=1`. So something like this should fix this:

```py
def get_random(request):
    seed = request.GET.get("seed", six.text_type(time.time()))
    custom_random = Random()
    if six.PY3:
        custom_random.seed(seed, version=1)
    else:
        custom_random.seed(seed)
    return custom_random
```

Reply:
Okay, ignore what I said, turns out there's some PhD-level `random` usage here so I give up :D